### PR TITLE
NEW PlansTest for testMessagePersistenceAfterManualScaleDown

### PR DESCRIPTION
**Pull request by tkratky**

### CODE REVIEW

**Comment by tkratky** [comment]
Btw. Jenkins is green! :)

**Comment by tkratky** [comment]
tests implemented in this PR(+1 test without task):
https://github.com/EnMasseProject/enmasse/issues/1123
https://github.com/EnMasseProject/enmasse/issues/1021

File: systemtests/src/test/java/io/enmasse/systemtest/common/plans/PlansTest.java [hunk 1]
```diff
@@ -307,6 +309,184 @@ public void testGlobalSizeLimitations() throws Exception {
...
+        TestUtils.waitForNBrokerReplicas(
+                kubernetes,
+                messagePersistAddressSpace.getNamespace(), 1, queue, new TimeoutBudget(2, TimeUnit.MINUTES));
+        //test failed in command above ^, functionality of test code below wasn't verified :) !TODO
+
+        //receive messages
+        Future<List<Message>> recvResult = queueClient.recvMessages(queue.getAddress(), msgs.size());
+        assertThat("Incorrect count of messages received", recvResult.get(1, TimeUnit.MINUTES).size(), is(msgs.size()));
+    }
+
```
Comment (kornys) [inline systemtests/src/test/java/io/enmasse/systemtest/common/plans/PlansTest.java]: Please close the clients :)

Comment (tkratky) [reply systemtests/src/test/java/io/enmasse/systemtest/common/plans/PlansTest.java]: Aren't clients closed automatically in TestBase? https://github.com/EnMasseProject/enmasse/blob/master/systemtests/src/test/java/io/enmasse/systemtest/bases/TestBase.java#L104
This shouldn't be different case and "default" tearDown should cover it. Correct me if I'm wrong please.

**Pull request merged**